Cargo.toml

```diff
@@ -6,6 +6,8 @@ members = [
     "common",
 ]
 
+resolver = "2"
```

Comment:
This not being explicitly specified caused the warning:
```
[nix-shell:~/code/birdseed-escrow/client]$ cargo run
warning: virtual workspace defaulting to `resolver = "1"` despite one or more workspace members being on edition 2021 which implies `resolver = "2"`
note: to keep the current resolver, specify `workspace.resolver = "1"` in the workspace root's manifest
note: to use the edition 2021 resolver, specify `workspace.resolver = "2"` in the workspace root's manifest
note: for more details see https://doc.rust-lang.org/cargo/reference/resolver.html#resolver-versions
```
Version 2 is the correct (newer) one